versioned_docs/version-2.x/magento2/installation.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
+title: Installation
+description:
+  Front-Commerce applications requires a Front-Commerce module to be installed
+  in Magento2. This guide explains how to install and configure it.
+---
+
+<p>{frontMatter.description}</p>
+
+## Module compatibility
+
+The Magento2 module is compatible with Magento2 2.3.2 to 2.4.5-p1 (Open Source,
+Commerce and B2B). To use respectively Commerce or B2B specific features, you
+need to install and setup [the Front-Commerce's Magento2 Commerce
+module](/docs/2.x/magento2/commerce#magento2-commerce-module-installation)
+or [the Front-Commerce's Magento2 B2B
+module](/docs/2.x/magento2/b2b#requirements).
```

Comment:
**issue:** we try to have a single source of truth for versions compatibility: the "Release notes" page.

**suggestion:** replace versions with a link to that page

```suggestion
The Magento2 module is compatible with Magento2 Open Source,
Commerce and B2B (see supported versions [in our release notes](/docs/2.x/appendices/release-notes#latest-version)). To use respectively Commerce or B2B specific features, you
need to install and setup [the Front-Commerce's Magento2 Commerce
module](/docs/2.x/magento2/commerce#magento2-commerce-module-installation)
or [the Front-Commerce's Magento2 B2B
module](/docs/2.x/magento2/b2b#requirements).
```